kuryr_kubernetes/controller/handlers/loadbalancer.py

```diff
@@ -435,6 +435,12 @@ def _remove_unused_members(self, loadbalancer_crd):
                     'pool_id']) in current_targets):
                 continue
 
+            # NOTE(maysams): Remove only lb member created during bootstrap
```

Comment:
Okay, I don't like it but whatever. Can you just replace the comment with some more detailed explanation? Like:

```
            # NOTE(maysams): The Endpoints object of kubernetes service is
            #                unstable during installation, so to minimize
            #                effect of this we're only removing node once
            #                we're sure it's gone from the deployment. That's
            #                done by looking for host-networking pods with
            #                that IP.
```